virtualizarr/kerchunk.py

```diff
@@ -302,6 +303,7 @@ def variable_to_kerchunk_arr_refs(var: xr.Variable, var_name: str) -> KerchunkAr
             shape=np_arr.shape,
             dtype=np_arr.dtype,
             order="C",
+            fill_value=None
```

Comment:
`fill_value` is currently required. My other PR is changing that to be a default of None.